pkg/bytecode/vm_test.go

```diff
@@ -479,6 +555,8 @@ func compileBytecode(t *testing.T, input string) *Bytecode {
 func makeValue(t *testing.T, a any) value {
 	t.Helper()
 	switch v := a.(type) {
+	case []any:
```

Comment:
This is ok for now, but you'll probably need to be more rigid in your type handling here. If the value should be `[]num` then you should be accepting `[]int` or `[]float64`. Once you get proper `any` wrapping, you'll treat an input Go type of `[]any` differently to `[]int` for example.

It will also make it catch errors in your test data at compile time rather at test run time.